ui/branding/templates/header.html

```diff
@@ -1,6 +1,47 @@
 {% extends "oarepo_ui/header.html" %}
 
 {% block navbar_search %}
-    <Searchbar search_link={"/docs"} />
+  <div class="item p-0 search-bar">
+  <div id="header-search-bar">
+    <form action="/docs" class="ui form" role="search">
+      <div class="ui fluid action input">
+        <input type="text" name="q" class="form-control" placeholder="{{ _('Search') }}">
+        <button type="submit" class="ui icon search button" aria-label="{{ _('Search') }}"><i class="search icon"></i></button>
+      </div>
+    </form>
+  </div>
+</div>
 {% endblock navbar_search %}
+{#Not sure if this should be uncommented in oarepo UI as probably not all repos will
+have communities/dashboard#}
```

Comment:
This could be safely moved to oarepo-ui

Reply:
@mirekys done in https://github.com/oarepo/oarepo-ui/pull/146